README.md

```diff
@@ -55,7 +55,7 @@ mkdir -p rendered/General/
 bin/jsonnet -J ./templates/vendor templates/General/ocp-performance-v2.jsonnet > rendered/General/ocp-performance-v2.json
 Rendered the v2 dashboards with latest grafonnet library
 ```
-All other operations remain same as before.
+Rest all operations reamin same as before.
```

Comment:
Not following this line. also, remain.

Reply:
Its just an update from current master branch. Corrected the typo.